app/views/includes/result-item.nunjucks

```diff
@@ -0,0 +1,60 @@
+{% if service.distance %}
+  <p class="results__address results__address-distance">
+  Distance: {{ service.distance | float | round(2) }} miles
+  </p>
+{% endif %}
+
+<h3 class="results__name">
+  {{ service.name }}
+</h3>
+
+{% if service.address %}
+  <p class="results__address results__address-lines">
+    {% if service.address.addressLines %} {{ service.address.addressLines | safe | replace(",", ", ") }}, {% endif %}
```

Comment:
Could the processing of `addressLines | safe ...` be done before the view please.